src/tr2/game/objects/creatures/cultist_1.c

```diff
@@ -0,0 +1,311 @@
+    CULTIST_1_STATE_AIM_1   = 6,
+    CULTIST_1_STATE_SHOOT_1 = 7,
+    CULTIST_1_STATE_AIM_2   = 8,
+    CULTIST_1_STATE_SHOOT_2 = 9,
+    CULTIST_1_STATE_AIM_3   = 10,
+    CULTIST_1_STATE_SHOOT_3 = 11,
+    CULTIST_1_STATE_DEATH   = 12,
+    // clang-format on
+} CULTIST_1_STATE;
+
+typedef enum {
+    CULTIST_1_ANIM_DEATH = 20,
+} CULTIST_1_ANIM;
+
+static const BITE m_Cultist1Gun = {
+    .pos = { .x = 3, .y = 331, .z = 56 },
+    .mesh_num = 10,
+};
+
+void Cultist1_Setup(void)
+{
+    OBJECT *const obj = &g_Objects[O_CULT_1];
+    if (!obj->loaded) {
+        return;
+    }
+
+    obj->initialise = Cultist1_Initialise;
+    obj->control = Cultist1_Control;
+    obj->collision = Creature_Collision;
+
+    obj->hit_points = CULTIST_1_HITPOINTS;
+    obj->radius = CULTIST_RADIUS;
+    obj->shadow_size = UNIT_SHADOW / 2;
+    obj->pivot_length = 50;
+
+    obj->intelligent = 1;
+    obj->save_anim = 1;
+    obj->save_position = 1;
+    obj->save_hitpoints = 1;
+    obj->save_flags = 1;
+
+    g_AnimBones[obj->bone_idx] |= BF_ROT_Y;
+}
+
+void Cultist1A_Setup(void)
+{
+    OBJECT *const obj = &g_Objects[O_CULT_1A];
+    if (!obj->loaded) {
+        return;
+    }
+
+    ASSERT(g_Objects[O_CULT_1].loaded);
+    obj->frame_base = g_Objects[O_CULT_1].frame_base;
+    obj->anim_idx = g_Objects[O_CULT_1].anim_idx;
+
+    obj->initialise = Cultist1_Initialise;
+    obj->control = Cultist1_Control;
+    obj->collision = Creature_Collision;
+
+    obj->hit_points = CULTIST_1_HITPOINTS;
+    obj->radius = CULTIST_RADIUS;
+    obj->shadow_size = UNIT_SHADOW / 2;
+    obj->pivot_length = 50;
+
+    obj->intelligent = 1;
+    obj->save_anim = 1;
+    obj->save_position = 1;
+    obj->save_hitpoints = 1;
+    obj->save_flags = 1;
+
+    g_AnimBones[obj->bone_idx] |= BF_ROT_Y;
+}
+
+void Cultist1B_Setup(void)
+{
+    OBJECT *const obj = &g_Objects[O_CULT_1B];
+    if (!obj->loaded) {
+        return;
+    }
+
+    ASSERT(g_Objects[O_CULT_1].loaded);
+    obj->frame_base = g_Objects[O_CULT_1].frame_base;
+    obj->anim_idx = g_Objects[O_CULT_1].anim_idx;
+
+    obj->initialise = Cultist1_Initialise;
+    obj->control = Cultist1_Control;
+    obj->collision = Creature_Collision;
+
+    obj->hit_points = CULTIST_1_HITPOINTS;
+    obj->radius = CULTIST_RADIUS;
+    obj->shadow_size = UNIT_SHADOW / 2;
+    obj->pivot_length = 50;
+
+    obj->intelligent = 1;
+    obj->save_anim = 1;
+    obj->save_position = 1;
+    obj->save_hitpoints = 1;
+    obj->save_flags = 1;
+
+    g_AnimBones[obj->bone_idx] |= BF_ROT_Y;
+}
+
+void __cdecl Cultist1_Initialise(const int16_t item_num)
+{
+    ITEM *const item = Item_Get(item_num);
+    if (Random_GetControl() < 0x4000) {
+        item->mesh_bits &= ~0b00110000;
+    }
+    if (item->object_id == O_CULT_1B) {
+        item->mesh_bits &= ~0b00011111'10000000'00000000;
+    }
+}
+
+void __cdecl Cultist1_Control(const int16_t item_num)
+{
+    if (!Creature_Activate(item_num)) {
+        return;
+    }
+
+    __int16 v9; // ax
```

Comment:
This seems like a leftover from porting maybe?

Reply:
Thanks – fixed.